components/resources/wallet_strings.grdp

```diff
@@ -1190,6 +1190,8 @@
   <message name="IDS_BRAVE_WALLET_REVIEW_SHIELD" desc="Review shield button label">Review shield</message>
   <message name="IDS_BRAVE_WALLET_SHIELDING" desc="Shielding label">Shielding</message>
   <message name="IDS_BRAVE_WALLET_SHIELD_ZEC" desc="Shield ZEC button label">Shield ZEC</message>
+  <message name="IDS_BRAVE_WALLET_SHIELD_FUNDS" desc="Shield funds button label">Shield funds</message>
+  <message name="IDS_BRAVE_WALLET_SHIELDING_FUNDS_ALERT_DESCRIPTION" desc="Shielding funds alert description">You are sending funds from transparent to your shielded account.</message>
```

Comment:
Seems like something is missing here. From transparent account?

Reply:
Updated to `You are sending funds from your transparent account to your shielded account.`